compiler/src/value/group/targets/edwards_bls12.rs

```diff
@@ -142,12 +142,14 @@ impl EdwardsGroupType {
     }
 
     pub fn edwards_affine_from_single(number: String, span: Span) -> Result<EdwardsAffine, GroupError> {
-        if number.eq("1") {
-            return Ok(edwards_affine_one());
-        } else if number.eq("0") {
+        if number.eq("0") {
             return Ok(EdwardsAffine::zero());
         } else {
-            Self::edwards_affine_from_x_str(number, span.clone(), None, span)
+            let one = edwards_affine_one();
+            let number_value = Fp256::from_str(&number).map_err(|_| GroupError::n_group(number, span))?;
+            let result: EdwardsAffine = one.mul(&number_value);
```

Comment:
Is this a "`mul_constant`" operation under the hood?

Reply:
It calls this method: https://github.com/AleoHQ/snarkOS/blob/9ad3dab5e9fc8373f8d4f0a9d4dc2b95f953346c/curves/src/templates/twisted_edwards_extended/mod.rs#L283